### ISSUE TITLE
Plugin offline install

### ISSUE THREAD

**Issue by guoxun19** [samcday/logstash-output-kinesis#24]
Hi @samcday 

I try to install the plugin with offline method because my VM cannot connect to https://rubygems.org. 
My environment is: logstash 5.0.0, JDK 1.8.0

I try the following steps but failed, could you please help me?
1. download plugin zip file from github, extract it and upload the folder logstash-output-kinesis-master to VM
2. add one line to logstash Gemfile: gem "logstash-output-kinesis", :path => "/tmp/logstash-output-kinesis-master
3. run  #bin/logstash-plugin install --no-verify

Error info:
[root@ip-172-31-5-94 logstash-5.0.0]# bin/logstash-plugin install --no-verify
Installing...
Error Bundler::GemspecError, retrying 1/10
There was a Errno::ENOENT while loading logstash-output-kinesis.gemspec: 
No such file or directory - git from
  /tmp/logstash-output-kinesis-master/logstash-output-kinesis.gemspec:17:in `eval_gemspec'

How can I finish offline install? Thank you!
Xun

**Comment by samcday** [comment]
Looks like you don't have git installed. The gemspec currently [uses git to enumerate the list of files to build](https://github.com/samcday/logstash-output-kinesis/blob/master/logstash-output-kinesis.gemspec).

I think a better way of installing this offline would be to download the .gem file from RubyGems.org and install that. You can download it by hitting the [Download](https://rubygems.org/downloads/logstash-output-kinesis-5.1.0-java.gem) link on the [RubyGems page](https://rubygems.org/gems/logstash-output-kinesis).

Let me know how you go with that!

**Comment by guoxun19** [comment]
@samcday I installed the plugin by using the .gem file. Actually, I have tried .gem file before but get error info of something like "could not found git when running 'git ls-files' ". So I use the following steps to finish plugin installation.
1. install git
2. download .gem file and run  "$LS_HOME/bin/logstash-plugin install --local ${path_to_gem_file}/logstash-output-kinesis-5.1.0-java.gem"

Thank you for your help! And I will close the issue.
